service/src/main/kotlin/app/cash/backfila/client/interceptors/OkHttpClientSpecifiedHeadersInterceptor.kt

```diff
@@ -0,0 +1,26 @@
+package app.cash.backfila.client.interceptors
+
+import app.cash.backfila.client.HttpHeader
+import okhttp3.Interceptor
+import okhttp3.Response
+
+internal class OkHttpClientSpecifiedHeadersInterceptor internal constructor(
+  private val headers: List<HttpHeader>?,
```

Comment:
how come this is nullable? if it's null, we shouldn't create the interceptor

Reply:
Fair enough, I've switched it to be non-nullable. The calling code already behaved that way.